.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: release
+assignees: ''
+
+---
+
+- [ ] Start a new release branch:
+```bash
+$ git flow release start X.Y.Z
+```
+- [ ] Rotate `CHANGELOG.md` (following [Keep a Changelog](https://keepachangelog.com/) principles)
+- [ ] Ensure outstanding changes are committed:
+```bash
+$ git add .
+$ git commit -m "X.Y.Z"
+```
+- [ ] Publish the release branch:
+```bash
+$ git flow release publish X.Y.Z
+```
+- [ ] Test the release branch on staging
+    - Wait for CI to cycle the release branch onto staging
+    - Thoroughly vet new features
+    - Commit any last-minute fixes
+- [ ] Start a new [release pipeline job](http://civicci01.internal.azavea.com/view/oar/job/Open%20Apparel%20Registry%20Release%20Pipeline/build?delay=0sec) with the SHA of `release/X.Y.Z` that was tested on staging
```

Comment:
Consider directing the person following the checklist to where they can find the correct SHA.

Reply:
Agreed. What do you think of how I resolved this?

Alternative approaches could be to check the latest Roller notification in **#oar**, or run `$ git rev-parse --short HEAD` after fetching the latest changes in `release/X.Y.Z`. 

Not sure if OAR has a `version.txt` file accessible, but like MMW, that could also work here.